softdev.bib

```diff
@@ -2060,3 +2060,39 @@ @article{boehm1988garbage
     publisher={Wiley Online Library},
     month=Sep
 }
+
+@article{boehm1993space,
+    author = {Boehm, Hans-Juergen},
+    title = {Space Efficient Conservative Garbage Collection},
+    journal = {SIGPLAN Not.},
+    volume = {28},
+    number = {6},
+    month = Jun,
+    pages = {197--206},
+    year = {1993},
+    publisher = {ACM},
+}
+
+@article{blackburn2003ulterior,
+    author = {Blackburn, Stephen M. and McKinley, Kathryn S.},
+    title = {Ulterior Reference Counting: Fast Garbage Collection Without a Long Wait},
+    journal = {SIGPLAN Not.},
+    volume = {38},
+    number = {11},
+    month = Oct,
+    year = {2003},
+    pages = {344--358},
+    publisher = {ACM},
+}
+
+@article{blackburn2004myths,
+    author = {Blackburn, Stephen M. and Cheng, Perry and McKinley, Kathryn S.},
+    title = {Myths and Realities: The Performance Impact of Garbage Collection},
+    journal = {SIGMETRICS Perform. Eval. Rev.},
```

Comment:
I'm not sure what conference this was in -- "International Conference on Measurement and Modeling of Computer Systems"?